pkg/publisher/publisher.go

```diff
@@ -227,7 +227,7 @@ func (p *cloudWatchPublisher) getCloudWatchMetricNamespace() *string {
 	return aws.String(cloudwatchMetricNamespace)
 }
 
-func getClusterID(ec2Client *ec2wrapper.EC2Wrapper) string {
+func getClusterID(ec2Client *ec2wrapper.EC2Wrapper, log logger.Logger) string {
```

Comment:
nit: maybe just make this a method function of cloudWatchPublisher if it's not used elsewhere